ui/src/jobs/details/EnsemblingJobDetailsView.js

```diff
@@ -31,47 +35,47 @@ export const EnsemblingJobDetailsView = ({ projectId, jobId, ...props }) => {
   const hasInitiallyLoaded = useInitiallyLoaded(isLoaded);
 
   return (
-    <EuiPage>
-      <EuiPageBody>
-        {!hasInitiallyLoaded ? (
-          <EuiFlexGroup direction="row">
-            <EuiFlexItem grow={true}>
-              <EuiLoadingContent lines={3} />
-            </EuiFlexItem>
-          </EuiFlexGroup>
-        ) : error ? (
-          <EuiCallOut
-            title="Sorry, there was an error"
-            color="danger"
-            iconType="alert">
-            <p>{error.message}</p>
-          </EuiCallOut>
-        ) : (
-          <Fragment>
-            <EuiPageHeader>
-              <EuiPageHeaderSection>
-                <PageTitle
-                  icon="sqlApp"
-                  iconSize="l"
-                  size="s"
-                  title={jobDetails.name}
-                  prepend={
-                    <StatusBadge
-                      status={JobStatus.fromValue(jobDetails.status)}
-                    />
-                  }
-                />
-              </EuiPageHeaderSection>
-            </EuiPageHeader>
-
+    <EuiPageTemplate restrictWidth={restrictWidth} paddingSize={paddingSize}>
+      <EuiSpacer size="l" />
```

Comment:
Instead of trying to hack the default CSS styles to make the page sections having the desired spacing, I found a way to use `EuiSpacer`s to create the same effect:

![Screenshot 2022-08-31 at 4 35 06 PM](https://user-images.githubusercontent.com/36802364/187635480-6fa2e4a9-4292-4704-9416-4662e3062a2c.png)

This will probably remove the need to overwrite any existing styling configs.